src/server.c

```diff
@@ -4622,7 +4622,7 @@ int finishShutdown(void) {
         rsiptr = rdbPopulateSaveInfo(&rsi);
         /* Keep the page cache since it's likely to restart soon */
         if (rdbSave(REPLICA_REQ_NONE, server.rdb_filename, rsiptr, RDBFLAGS_KEEP_CACHE) != C_OK) {
-            /* Ooops.. error saving! The best we can do is to continue
+            /* Oops.. error saving! The best we can do is to continue
```

Comment:
Some projects are "cute". This one didn't feel particularly cute, and as such I offer the spelling fix here. Sometimes I'd also fix up the number of dots from two to three, but I didn't. I'm happy to add the extra dot...